messages.md

```diff
@@ -65,7 +72,8 @@ participants.
             ...
         },
         "txid": "txid",
-        "deposit_outpoint": "deposit utxo outpoint"
+        "deposit_outpoint": "deposit utxo outpoint",
+        "derivation_index": 42
```

Comment:
is it always gonna be 42?

Reply:
nit: I think it'd be clearer to state data types or specify the range of values

Reply:
I agree but i didn't find a clean and less confusing solution :/